pootle/apps/pootle_store/views.py

```diff
@@ -403,9 +403,7 @@ def get_units(request):
             # Django looks for `sort_by_field` field in the initial table.
             # https://code.djangoproject.com/ticket/19434
             uid_list = [u['id'] for u
-                        in step_queryset.values('id', 'sort_by_field',
-                                                'store__pootle_path')]
-
+                        in step_queryset.values('id', 'sort_by_field')]
```

Comment:
`extra` seems not to be used anymore, so perhaps also adjust comment above and just retrieve the `id` with a `values_list('id', flat=True)`.